content/howto/geodesic/authenticate-with-leapp.md

```diff
@@ -0,0 +1,182 @@
+---
+title: "Authenticate with AWS using Leapp"
+description: "Learn how to use Leapp to supply AWS credentials to tools used within Geodesic."
+weight: 1
+---
+## Intro
+
+In this how-to, we will help you get started using Geodesic to work with AWS
+resources by helping you set up and use [Leapp](https://leapp.cloud) to handle 
+credentials and authentication.
```

Comment:
```suggestion
credentials and authentication. Leapp is an open-source tool that makes this easier.
```
I feel like we should point this out because the landing page makes me feel like it has a SaaS component.